src/PrometheusPushGateway/PushGateway.php

```diff
@@ -39,50 +42,50 @@ public function __construct($address, ClientInterface $client = null)
      * Uses HTTP PUT.
      * @param CollectorRegistry $collectorRegistry
      * @param string $job
-     * @param array $groupingKey
+     * @param array<string> $groupingKey
```

Comment:
we ended up with `string[]` and `array<string>` i think I'll try to update the other project to use the later syntax as well.

Reply:
Actually, this is a quite "special" case. It needs to be an assoc string array, therefore i decided to use this notation. Within the PHP client, it is an "array" with strings as values.

Reply:
@rdohms can i resolve this or should i change it to another notation?

Reply:
LGTM